docs/sources/alerting/manage-notifications/template-notifications/_index.md

```diff
@@ -11,7 +11,7 @@ weight: 100
 
 # Customize notifications
 
-Customize your notifications by creating message templates to use in the notifications sent to contact points.
+Customize your notifications with creating message templates.
```

Comment:
```suggestion
Customize your notifications by creating message templates.
```

Reply:
Can we then add something to this like: Notifications are then sent to your contact point integrations, such as Slack or email.

Reply:
I think it'd help explaining the flow in both places